### ISSUE TITLE
[no-magic-numbers] False positive for readonly properties

### ISSUE THREAD

**Issue by mrlika** [typescript-eslint/typescript-eslint#934]
```JSON
{
  "rules": {
    "@typescript-eslint/no-magic-numbers": ["error"]
  }
}
```

```TS
class IdbDatabase {
    public static readonly DATABASE_VERSION = 1;
}
```

**Expected Result**

No error

**Actual Result**

error  No magic number: 1  @typescript-eslint/no-magic-numbers

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `2.0.0` |
| `@typescript-eslint/parser`        | `2.0.0` |
| `TypeScript`                       | `3.5.3` |
| `ESLint`                           | `6.2.0` |
| `node`                             | `11.14.0` |
| `npm`                              | `6.7.0` |
